website/docs/error-reference.md

```diff
@@ -2858,6 +2858,12 @@ Sorbet treats the `::Class` and `::Module` classes in the Ruby standard library
 
 Note that this error is only reported in `# typed: strict` files and above. Downgrading the file that declares an explicit subclass of `Module` to `typed: true` will silence this error. (This also applies to direct subclasses of `Class`, but note that directly subclassing `Class` is rejected by the Ruby VM at runtime.)
 
+## 5082
+
+> This error is specific to Stripe's custom `--sorbet-packages` mode. If you are at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+When running in package-directed mode, Sorbet requires that classes and modules not be reopened once their package has been type checked. This error indicates that a class or module has been re-opened for the purpose of adding a mixin.
```

Comment:
Is there any chance you could be more constructive?

The error reference should ideally be the way to say what to do instead. When the error reference just restates what the error message already said but more verbosely, it trains people that there's not that much value in following a link to an error code.

It's not the case for most of our packager errors, which is a problem I would like to get away from. But for a lot of our other errors, they're more constructive.